spec/yardstick/v2_client/grant_status_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Yardstick::V2Client::GrantStatus do
+  let(:gs_client) { Yardstick::V2Client::GrantStatus }
+  let(:token) { 'abssclklkjlkj' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.